client/requestUtil.js

```diff
@@ -108,6 +108,7 @@ RequestUtil.prototype.saveAWSCredentials = function (parsedResponse) {
 
   this.sqs = parsedResponse.sqs
   this.sns = parsedResponse.sns
+  this.initialSyncDone = true
```

Comment:
minor: it would be more semantic if this variable were named `listInProgress` because we don't set the var until we call `listObjects()`.

also, initializing `initialSyncDone = true` doesn't make sense in the case we didn't do a sync yet, as we quickly set it to `false`. in case we keep `initialSyncDone`, maybe we should init as `undefined`, then check within `listObjects` whether `initialSyncDone === false` (rather than falsey).